test/sanity/validate-modules/validate-modules

```diff
@@ -901,6 +931,7 @@ class ModuleValidator(Validator):
             )
 
 
+
```

Comment:
This line will create a pep8/flake8 failure.

Unfortunately this file, not having a `.py` extension currently, isn't automatically picked up in pep8.  I'll be changing that soon.